deploy/10_deploy_risk_manager_v1.ts

```diff
@@ -1,6 +1,11 @@
-module.exports = async ({ getNamedAccounts, deployments }) => {
```

Comment:
Can't add comment to the proper lines, because there were no changes in them so I add this here, on top of the file.

Lines 16-19
```
const auctionLength = 86400 // 24h
  const bondAuctionThreshold = 100
  const initialSwapStrategy =
    process.env.INITIAL_SWAP_STRATEGY || "SignerBondsManualSwap"
```

I think since we use TypeScript here we should get used to declaring the types for the variables that we declare even if we are setting the value for them right away (see examples in the [documentation](https://www.typescriptlang.org/docs/handbook/basic-types.html).

So i would change this code to:
```
const auctionLength: number: number = 86400 // 24h
  const bondAuctionThreshold: number = 100
  const initialSwapStrategy: string =
    process.env.INITIAL_SWAP_STRATEGY || "SignerBondsManualSwap"
```

I think we do not need to specify the types for the variables that comes from `hre` and its properties, grand-properties etc. (although we probably if you think it will improve readability) since `HardhatRuntimeEnvironment` take care of that (but please see my first comment about `tsconfig.json`)

Thoughts?

Reply:
I think that we could also specify the units for `bondAuctionThreshold` in the comment. For `auctionLength` it is clear that the unit is in seconds since we have information that it's 24h. For `bondAuctionThreshold` it is not so clear IMO.

Reply:
fdde04b83c6a6d6c8f0e12af8a65240d645cd6d1